src/test/java/com/facebook/openwifirrm/optimizers/tpc/RandomTxPowerInitializerTest.java

```diff
@@ -48,14 +49,36 @@ void test1() throws Exception {
 		dataModel.latestState.put(deviceA, createState());
 		dataModel.latestState.put(deviceB, createState());
 
-		final int txPower = 16;
+		// Test providing tx power
+		for (int txPower: Arrays.asList(-1, 16, 27)) {
```

Comment:
style: missing space before `:`

Reply:
Two more generic questions here: 
1. Do we have a doc for the Java formatting standard we are using?
2. Do we have any linter to auto format our code?

Reply:
No to both (sorry!)